### PR TITLE
Div 2304 reassure users contact details wont be shared develop

### PR DESCRIPTION
**Pull request by niyioyelade**
# Description

https://tools.hmcts.net/jira/browse/DIV-2304

## Type of change

Confidentiality assurance message included.

# How Has This Been Tested?

Unit tests and manual testing

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/steps/petitioner/correspondence/address/template.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{% extends "../../../../../app/components/AddressLookupStep/template.html" %}
+
+{% block warning %}
+  {% if session.petitionerContactDetailsConfidential != 'share' %}
+		<p class="panel panel-border-wide">{{content.confidentialAddressMessage | safe}}</p>
+	{% endif %}
```
Comment (dtaylor7) [inline app/steps/petitioner/correspondence/address/template.html]: can you fix the formatting here ( remove extra spaces )

File: app/steps/petitioner/home/address/template.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{% extends "../../../../../app/components/AddressLookupStep/template.html" %}
+
+{% block warning %}
+  {% if session.petitionerContactDetailsConfidential != 'share' %}
+		<p class="panel panel-border-wide">{{content.confidentialAddressMessage | safe}}</p>
+	{% endif %}
```
Comment (dtaylor7) [inline app/steps/petitioner/home/address/template.html]: fix formatting here too

File: app/steps/petitioner/home/address/template.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{% extends "../../../../../app/components/AddressLookupStep/template.html" %}
```
Comment (dtaylor7) [inline app/steps/petitioner/home/address/template.html]: could you try with single dot notation as suggested here https://github.com/mozilla/nunjucks/issues/13#issuecomment-18854022.

so possibly `{% extends "./../components/AddressLookupStep/template.html" %}` would work

File: app/steps/petitioner/home/address/template.html (hunk 2 above)
Comment (niyioyelade) [reply app/steps/petitioner/home/address/template.html]: Done

File: app/steps/petitioner/correspondence/address/template.html (hunk 1 above)
Comment (niyioyelade) [reply app/steps/petitioner/correspondence/address/template.html]: Done

**Pull request merged**